x-pack/plugins/canvas/server/templates/theme_dark.ts

```diff
@@ -0,0 +1,397 @@
+      },
+    },
+  },
+};
```

Comment:
These previously existed, and here they are just moved to `/server` and also converted from plain json to ts so that they are properly typed, which will help if workpad the workpad type changes in the future.  So these are not ending up in the bundle.